src/ui/src/app/chart-details/chart-details.component.scss

```diff
@@ -3,31 +3,95 @@
 
 .chart-details {
   width: 100%;
-  margin: 3em 0;
-  padding: 0 2em;
   display: flex;
-  flex-direction: column-reverse;
-  flex-wrap: wrap;
+  flex-direction: column;
 
-  @include mappy-bp(medium) {
-    flex-direction: row;
-    flex-wrap: nowrap;
-
-    &__info {
-      padding-left: 1em;
+  &__header {
+    width: 100%;
+    margin-top: -80px;
+    margin-bottom: 40px;
+    &__background {
+      width: 100%;
+      height: 200px;
+      background: md-color($monocular-app-primary);
```

Comment:
This color will be overridden by the Icon's color when it's calculated. This will cause a flashing effect in the UI.